core/message-tags-3.3.md

```diff
@@ -176,3 +179,19 @@ In this example, plus signs, colons, equals signs and commas are transmitted raw
 ```
 @+example=raw+:=,escaped\:\s\\ :irc.example.com NOTICE #channel :Message
 ```
+
+---
+
+An empty message sent by a client with the `+example-client-tag` client-only tag:
+
+```
+@+example-client-tag=example-value PRIVMSG #channel :
+```
+
+---
+
+An alternate form of an empty message sent by a client with the `+example-client-tag` client-only tag, where the last parameter is omitted
+
+```
+@+example-client-tag=example-value PRIVMSG #channel
```

Comment:
@DarthGandalf other than the mentioned use cases for content-less messages, is your objection that this is allowing `PRIVMSG #channel` rather than `PRIVMSG #channel :`?

I could change it to require the last param but allow it to be empty. But since it's allowing that param to effectively be ignored, I'm not sure it's needed.

---

I see the reason for allowing this change is so that servers can keep the standard PRIVMSG/NOTICE routing for message delivery to channels/users, and use them for pure tag delivery. Another alternative is to define a new message type that just carries tags, but since empty messages aren't allowed/useful normally, it makes sense to use them.